core/modules/simpletest/simpletest.module

```diff
@@ -369,16 +369,16 @@ function simpletest_test_get_all() {
         $classes = backdrop_parse_info_file($file->uri, TRUE);
         $error = '';
         if (empty($classes)) {
-          $error = 'File %uri contains no testing classes.';
+          $error = t('File @uri contains no testing classes.');
```

Comment:
Is that `@uri` supposed to get replaced by `$file->uri`? I'm asking because that replacement doesn't happen. I'm aware it also didn't happen before, but this might be the right time to fix it.